lib/tests/streamlit/vega_lite_test.py

```diff
@@ -160,47 +160,32 @@ def test_no_args_add_rows(self):
         c = self.get_delta_from_queue().new_element.vega_lite_chart
         self.assertEqual(len(c.data.data.cols[0].strings.data), 4)
 
-    def test_width_zero(self):
-        """Test that width=0 autosets to full width."""
-        st.vega_lite_chart(df1, {"mark": "rect"}, width=0)
+    def test_use_container_width(self):
+        """Test that use_container_width=True autosets to full width."""
+        st.vega_lite_chart(df1, {"mark": "rect"}, use_container_width=True)
 
         c = self.get_delta_from_queue().new_element.vega_lite_chart
         self.assertDictEqual(
             json.loads(c.spec), merge_dicts(autosize_spec, {"mark": "rect"})
         )
 
-    def test_width_positive(self):
-        """Test that width > 0 sets the width."""
-        st.vega_lite_chart(df1, {"mark": "rect"}, width=500)
-
-        c = self.get_delta_from_queue().new_element.vega_lite_chart
-        self.assertDictEqual(
-            json.loads(c.spec),
-            merge_dicts(autosize_spec, {"mark": "rect", "width": 500}),
-        )
-
-    def test_width_neg_one(self):
-        """Test that width=-1 leaves the width up to Vega-Lite."""
-        st.vega_lite_chart(df1, {"mark": "rect"}, width=-1)
-
-        c = self.get_delta_from_queue().new_element.vega_lite_chart
-        self.assertDictEqual(json.loads(c.spec), {"mark": "rect"})
+        self.assertEqual(c.use_container_width, True)
 
     def test_width_inside_spec(self):
-        """Test that {width:-1} leaves the width up to Vega-Lite."""
```

Comment:
Should we update the name of the test? 

We're not setting the width inside the spec anymore it seems.